release_images_patches.yml

```diff
@@ -235,9 +235,36 @@ release_images:
       example: False
       disable_sm_tag: False
       force_release: False
-
-  # TensorFlow 2.16 Images
+  
+  # PyTorch 2.5 Images
   19:
+    framework: "pytorch"
+    version: "2.5.1"
+    arch_type: "x86"
+    customer_type: "ec2"
+    inference:
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py311" ]
+      os_version: "ubuntu22.04"
+      cuda_version: "cu124"
+      example: False
+      disable_sm_tag: False
+      force_release: False
+  20:
+    framework: "pytorch"
+    version: "2.5.1"
+    arch_type: "x86"
+    customer_type: "sagemaker"
+    inference:
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py311" ]
+      os_version: "ubuntu22.04"
+      cuda_version: "cu124"
+      example: False
+      disable_sm_tag: False
+      force_release: False
+  # TensorFlow 2.16 Images
```

Comment:
nit: can you add a blank line here just for organization.

Reply:
Sure. Done.